server/internal/server/completions.go

```diff
@@ -127,7 +127,13 @@ func (s *S) CreateChatCompletion(
 			IncludeUsage: true,
 		}
 	}
-
+	// max_tokens is deprecated and replaced by max_completion_tokens, now
+	if createReq.MaxCompletionTokens == 0 {
```

Comment:
How about also having `createReq.MaxTokens = createReq.MaxCompletionTokens` when `createReq.MaxTokens == 0` and adding a (TODO) comment mentioning that this is required as Ollama doesn't support `MaxCompletionTokens` as of Feb 2025?

Reply:
good idea!